extension/src/main/webapp/WEB-INF/conf/gamification/search/rule-search-query.json

```
@@ -14,7 +14,7 @@
         {
           "query_string": {
             "fields": ["title", "description"],
-            "query": "@term_query@",
+            "query": "*@term_query@*",
```

```suggestion
            "query": "@term_query@",
```
Nope, we shouldn't make the search so permissive to be able to search with a letter (like we do with other indices)

---

In fact if you would like to make the search more permissive, you can change the fuzziness index.